forum/qa-content/qa-ask.js

```diff
@@ -296,3 +296,36 @@ function set_category_description(idprefix)
 		n.innerHTML=desc;
 	}
 }
+
+/*
+ * Feature - Remove unnecessary # from tags, when creating new post (question)
+ */ 
+ ;(function (document)
+ {
+	 'use strict';
+	 
+	 // wait for DOM to load
+	 window.addEventListener('DOMContentLoaded', function()
+	 {
+		// get DOM elements: "tags" input and button "Zadaj pytanie"
+		var tags = document.getElementById('tags');
+		var askQuestionBtn = document.querySelector('input[value="Zadaj pytanie"]');
+		
+		// when user clicks "Zadaj pytanie"
+		askQuestionBtn.addEventListener('click', function(ev)
+		{					
+			// get written tags and split them
+			var allTags =  tags.value.split(' ');
+			
+			// filter tags, accept only those without # and those like "C#"
+			var filteredTags = allTags.filter(function(hash)
+			{
+				if (hash.indexOf('#') < 0 || hash.toLowerCase().indexOf('c#') > -1)

```

Comment:
Better to put omitting languages in array. 
What do you think? 
we have C# F# ..
https://en.wikipedia.org/wiki/List_of_programming_languages